core/backend/src/test/integration/ChangedElements.test.ts

```diff
@@ -148,8 +150,9 @@ describe("ChangedElements (#integration)", () => {
     assert.isTrue(changeData!.changedElements.elements.length === changeData!.changedElements.classIds.length);
     assert.isTrue(changeData!.changedElements.elements.length === changeData!.changedElements.opcodes.length);
     assert.isTrue(changeData!.changedElements.elements.length === changeData!.changedElements.type.length);
-    // assert.isTrue(changeData?.changedElements.elements.length === changeData!.changedElements.properties!.length);
-    // NEEDS_WORK: DP
+    assert.isTrue(changeData?.changedElements.elements.length === changeData!.changedElements.properties!.length);
+    assert.isTrue(changeData?.changedElements.elements.length === changeData!.changedElements.oldChecksums!.length);

```

Comment:
@diegopinate this test will fail on master until new addon containing your changes is integrated, won't it? If so this PR should be merged into the imodel02 branch, not master.

Reply:
When merging with imodel02 branch please follow [these instructions](https://dev.azure.com/bentleycs/iModelTechnologies/_wiki/wikis/iModelTechnologies.wiki/8690/iModel.js-Node-add-on-development) carefully.